presto-hive/src/main/java/io/prestosql/plugin/hive/util/HiveFileIterator.java

```diff
@@ -51,23 +52,26 @@
     private final DirectoryLister directoryLister;
     private final NamenodeStats namenodeStats;
     private final NestedDirectoryPolicy nestedDirectoryPolicy;
+    private final boolean ignoreAbsentPartitions;
 
-    private Iterator<LocatedFileStatus> remoteIterator = Collections.emptyIterator();
+    private Iterator<LocatedFileStatus> remoteIterator = emptyIterator();
```

Comment:
It used to use `Collections.emptyIterator()`, now it uses `emptyIterator`. 
This looks like a non intentional change, please revert.